seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/local/LocalFileIT.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.e2e.connector.file.local;
+
+import org.apache.seatunnel.e2e.common.TestSuiteBase;
+import org.apache.seatunnel.e2e.common.container.TestContainer;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.TestTemplate;
+import org.testcontainers.containers.Container;
+
+import java.io.IOException;
+
+public class LocalFileIT extends TestSuiteBase {
+
+    @TestTemplate
+    public void testLocalFileWrite(TestContainer container) throws IOException, InterruptedException {
+        Container.ExecResult textResult = container.executeJob("/text/fake_to_local_file_text.conf");
+        Assertions.assertEquals(0, textResult.getExitCode());
+        Container.ExecResult jsonResult = container.executeJob("/json/fake_to_local_file_json.conf");
+        Assertions.assertEquals(0, jsonResult.getExitCode());
+        Container.ExecResult orcResult = container.executeJob("/orc/fake_to_local_file_orc.conf");
+        Assertions.assertEquals(0, orcResult.getExitCode());
+        Container.ExecResult parquetResult = container.executeJob("/parquet/fake_to_local_file_parquet.conf");
```

Comment:
Add `local_file_xxx_to_assert.conf` validate row count & field types?

Reply:
Yup, this pull request is not completed. It's also a draft pr.